src/index.js

```diff
@@ -0,0 +1,55 @@
+const tempVal = document.getElementById('value');
+const condition = document.getElementById('condition');
+const city = document.getElementById('city');
+const cardWiki = document.getElementById('cardWiki');
+const cardSnippet = document.getElementById('snippet');
+const cardTitle = document.getElementById('cityName');
+const toggle = document.getElementById('switch');
+
```

Comment:
- [ ] Kindly separate the dom-manipulation tasks from the application logic. This will keep your code clean and allow you to create different files for different tasks which is always good practice.  Since you have already learned about modules in javascript, kindly use that power to separate your code. ⬇️ 
![dom_logic_separation_requirement](https://user-images.githubusercontent.com/32361963/111024167-43548600-83ee-11eb-8521-abf485944eb0.png)